sdk/nodejs/dagger/api/utils.ts

```diff
@@ -0,0 +1,42 @@
+  export function queryBuilder(q: any) {
```

Comment:
Is this file generated dynamically too or it will be a static utils file?

Reply:
This file will be static, not generated.